climada/util/test/test_yearsets.py

```diff
@@ -0,0 +1,111 @@
+"""
+This file is part of CLIMADA.
+
+Copyright (C) 2017 ETH Zurich, CLIMADA contributors listed in AUTHORS.
+
+CLIMADA is free software: you can redistribute it and/or modify it under the
+terms of the GNU Lesser General Public License as published by the Free
+Software Foundation, version 3.
+
+CLIMADA is distributed in the hope that it will be useful, but WITHOUT ANY
+WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+PARTICULAR PURPOSE.  See the GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License along
+with CLIMADA. If not, see <https://www.gnu.org/licenses/>.
+
+---
+
+Test of dates_times module
+"""
+
+import unittest
+import numpy as np
+import collections
+
+import climada.util.yearsets as yearsets
+from climada.engine import Impact
+import climada.util.dates_times as u_dt
+
+
+EVENT_IMPACTS = Impact()
+EVENT_IMPACTS.at_event = np.arange(10,110,10)
+EVENT_IMPACTS.frequency = np.array(np.ones(10)*0.2)
+
+SAMPLING_DICT = dict()
+SAMPLING_DICT = {'selected_events': np.tile(np.arange(0,10),2),
+                  'events_per_year': np.array(np.ones(10, dtype=int)*2)}
+
+N_SAMPLED_YEARS = 10
+YEAR_LIST = np.arange(2000, 2010).tolist()
```

Comment:
I guess it makes sense to write this directly as a list if you are going to turn it to a list after? list(range(2000, 2010))
there are a few of these below as well :)